packages/wrangler/src/__tests__/cloudchamber/apply.test.ts

```diff
@@ -2026,10 +1997,9 @@ describe("cloudchamber apply", () => {
 			├ EDIT my-container-app
 			│
 			│     [containers.configuration]
-			│     image = \\"${registry}/some-account-id/hello:1.0\\"
+			│     image = \\"registry.cloudflare.com/some-account-id/hello:1.0\\"
```

Comment:
Is this intended?

Reply:
PS: I just checked and saw that we do have the registry address like this here: https://github.com/cloudflare/workers-sdk/blob/2e5b8802b3d9b9b8ff0d44c6b7a5d80e64f92313/packages/wrangler/src/__tests__/containers/deploy.test.ts#L1324c